### PR TITLE
chore: prepare release

### PR DESCRIPTION
**Pull request by dasanra**
## Versions

    @safe-global/api-kit: v2.5.0
    @safe-global/protocol-kit: v5.0.0
    @safe-global/relay-kit: v3.2.0
    @safe-global/sdk-starter-kit: 1.0.0
    @safe-global/testing-kit: 0.1.0
    @safe-global/types-kit: v1.0.0

## Main changes
* feat: Viem migration + Passkeys by @leonardotc in https://github.com/safe-global/safe-core-sdk/pull/931
* chore(deps): bump micromatch from 4.0.7 to 4.0.8 by @dependabot in https://github.com/safe-global/safe-core-sdk/pull/960
* chore(deps): bump path-to-regexp from 1.8.0 to 1.9.0 by @dependabot in https://github.com/safe-global/safe-core-sdk/pull/970
* feat: remove onramp-kit and auth-kit from repo by @dasanra in https://github.com/safe-global/safe-core-sdk/pull/972
* feat: Testing kit by @yagopv in https://github.com/safe-global/safe-core-sdk/pull/962
* feat(sdk-starter-kit): Improvements by @yagopv in https://github.com/safe-global/safe-core-sdk/pull/967
* chore: bump dependencies by @dasanra in https://github.com/safe-global/safe-core-sdk/pull/974
* chore: Types kit by @yagopv in https://github.com/safe-global/safe-core-sdk/pull/973
* fix(testing-kit): Issues with executable by @yagopv in https://github.com/safe-global/safe-core-sdk/pull/979
* chore: code cleanup and improvements by @dasanra in https://github.com/safe-global/safe-core-sdk/pull/977
* feat(protocol-kit): Predict Safe address improvements by @yagopv in https://github.com/safe-global/safe-core-sdk/pull/982
* feat(protocol-kit): Add deployment functionality to Safe class by @DaniSomoza in https://github.com/safe-global/safe-core-sdk/pull/980
* chore: set release versions by @dasanra in https://github.com/safe-global/safe-core-sdk/pull/992

### CODE REVIEW

**Review by yagopv — APPROVED** [review]

File: .github/workflows/protocol-kit-e2e-test.yml [hunk 1]
```diff
@@ -27,4 +27,5 @@ jobs:
       - name: Test ${{ matrix.provider }} - Safe ${{ matrix.contract-version }}
         run: |
           cd packages/protocol-kit
+          ls -la $(pwd)/node_modules/.bin
```
Comment (dasanra) [inline .github/workflows/protocol-kit-e2e-test.yml:30]: Not necessary anymore

File: packages/testing-kit/tsconfig.json [hunk 2]
```diff
@@ -4,5 +4,5 @@
     "composite": true,
     "outDir": "dist"
   },
-  "include": ["src/**/*"]
+  "include": ["bin/**/*", "src/**/*"]
```
Comment (dasanra) [inline packages/testing-kit/tsconfig.json:7]: `bin/**/+ shouldn't be necessary

File: packages/testing-kit/package.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@safe-global/testing-kit",
+  "version": "0.0.1-alpha.1",
```
Comment (dasanra) [inline packages/testing-kit/package.json:3]: Set release version to 0.1.0

File: packages/relay-kit/src/packs/safe-4337/estimators/PimlicoFeeEstimator.test.ts [hunk 4]
```diff
@@ -32,19 +32,23 @@ describe('PimlicoFeeEstimator', () => {
       entryPoint: fixtures.ENTRYPOINTS[0]
     })
 
-    expect(sponsoredGasEstimation).toEqual(fixtures.USER_OPERATION_GAS_PRICE.fast)
+    expect(sponsoredGasEstimation).toEqual({ maxFeePerGas: 100000n, maxPriorityFeePerGas: 200000n })
   })
 
   // TODO: This tests breaks because of the BigInt serialization and requires further investigation
```
Comment (dasanra) [inline packages/relay-kit/src/packs/safe-4337/estimators/PimlicoFeeEstimator.test.ts:38]: Remove TODO

**Pull request merged**